<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>771</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_cmd">
     <attribute name="title">
      <string>Tab_cmd</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_run">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>20</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_cmd">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>411</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_project">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>68</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Project</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_project">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_cmd">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Command</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_readoutput">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>741</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_stop">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>20</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_run_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_new">
     <attribute name="title">
      <string>Tab_new</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>765</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
